### DISCUSSION TITLE
create new docker container from saved image using different settings and DB

### DISCUSSION THREAD

**Discussion by merryo11** [Keyfactor/ejbca-ce#352] [General]
I have a docker container with ejbca-ce 8.0 and embedded H2 db. I commit the docker after basic installations with some certificate issuances and now want to use the saved image (after commit image) to connect with mysql fresh DB so can anyone guide me what command shall i use to connect my saved settings with new DB as a new container.

Note: mysql DB shall be on another machine while mysql client has been installed on the host of the machine which shall create the new container from comitted image.

**Comment by primetomas** [comment]
Migrating from an H2 database to a MySQL database is not completely trivial. I rather suggest that you start a new installation.
Otherwise you can follow something like this. https://stackoverflow.com/questions/11359813/how-to-convert-h2database-database-file-to-mysql-database-sql-file

The Docker Hub documentation has documentation how to point the container to a MySQL/MariaDB database.
https://hub.docker.com/r/keyfactor/ejbca-ce
